app/models/converters/app_converter.rb

```diff
@@ -19,8 +20,20 @@ def to_compose_yaml
       compose_hash.to_yaml
     end
 
+    def to_compose_gist(filename=DOCKER_COMPOSE_FILENAME)
+      github_client.create_gist({
+                                  description: "created by Panamax",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.